--- fann-1.1.0-old/python/Makefile	2004-06-06 00:33:09.817753008 +0200
+++ fann-1.1.0/python/Makefile	2004-06-06 00:32:31.565568224 +0200
@@ -6,10 +6,10 @@
 all: $(TARGETS)
 
 _%.so: %_wrap.o
-	gcc -shared $< -lfann -o $@ 
+	gcc -shared ${CFLAGS} $< -L../src/.libs/ -lfann -o $@ 
 
 %.o: %.c 
-	gcc -c -fpic $< -I/usr/include/python2.3/
+	gcc ${CFLAGS} -c -fpic $< -I/usr/include/python2.3/
 
 %_wrap.c: %.i 
 	swig -python $<
