diff -urN fann-2.0.0/python/pyfann/Makefile fann-2.0.0.new/python/pyfann/Makefile
--- fann-2.0.0/python/pyfann/Makefile	2005-12-03 02:22:43.000000000 +0100
+++ fann-2.0.0.new/python/pyfann/Makefile	2006-01-07 20:18:27.000000000 +0100
@@ -5,12 +5,12 @@
 
 PYTHON=python2.4
 FANN_DIR="../.."
-LIBS=-L. -L/usr/local/lib -L/usr/bin -l$(PYTHON) $(FANN_DIR)/src/fann*.o
+LIBS=-L. -L/usr/lib -L/usr/bin -l$(PYTHON) $(FANN_DIR)/src/fann*.o
 
 all: $(TARGETS)
 	
 _%.so: pyfann_wrap.o fann_helper.o
-	gcc $(LIBS) -shared -dll $^ -o $@
+	gcc $(LIBS) -shared $^ -o $@
 
 %.o: %.c 
 	gcc -c $< -I/usr/include/$(PYTHON)/ -I$(FANN_DIR)/src/include/
