diff -urN fann-1.2.0/python/makefile.gnu fann-1.2.0.new/python/makefile.gnu
--- fann-1.2.0/python/makefile.gnu	2004-10-09 13:56:57.000000000 +0200
+++ fann-1.2.0.new/python/makefile.gnu	2005-11-13 20:10:45.000000000 +0100
@@ -1,18 +1,18 @@
 # This makefile was written to compile a distribution of pyfann for
 # GNU platforms (cygwin included.)
 
-TARGETS = _libfann.dll
+TARGETS = _libfann.so
 
-PYTHON=python2.3
-LIBS=-L. -L/usr/local/lib -L/usr/bin -l$(PYTHON) ../src/fann*.o
+PYTHON=python2.4
+LIBS=-L. -L/usr/lib -L/usr/bin -l$(PYTHON) ../src/fann*.o
 
 all: $(TARGETS)
 
-_%.dll: %_wrap.o fann_helper.o
-	gcc $(LIBS) -shared -dll $^ -o $@
+_%.so: %_wrap.o fann_helper.o
+	gcc $(LIBS) -shared $^ -o $@
 
 %.o: %.c 
-	gcc -c $< -I/usr/include/$(PYTHON)/
+	gcc -c $< -I/usr/include/$(PYTHON)/ -I../src/include/
 
 %_wrap.c: %.i 
 	swig -python $<
