--- fann-1.2.0/python/makefile.gnu.old	2005-08-03 11:08:19.000000000 +0200
+++ fann-1.2.0/python/makefile.gnu	2005-08-03 11:06:46.000000000 +0200
@@ -3,7 +3,7 @@
 
 TARGETS = _libfann.dll
 
-PYTHON=python2.3
+PYTHON=python2.4
 LIBS=-L. -L/usr/local/lib -L/usr/bin -l$(PYTHON) ../src/fann*.o
 
 all: $(TARGETS)
@@ -12,7 +12,7 @@
 	gcc $(LIBS) -shared -dll $^ -o $@
 
 %.o: %.c 
-	gcc -c $< -I/usr/include/$(PYTHON)/
+	gcc -c $< -I/usr/include/$(PYTHON)/ -I../src/include/
 
 %_wrap.c: %.i 
 	swig -python $<
